.github/workflows/Tests.yml

```
@@ -276,28 +185,26 @@ jobs:
-      run: |
-        if [ "$OS" == "ubuntu-latest" ] ; then (curl https://deepsource.io/cli | sh) || true ; else echo "SKIP deepsource" ; fi ;
     - name: Pre-Clean
       id: clean
       run: make -j1 -f Makefile clean || true ;
+    - name: Install code-climate tools for ${{ matrix.python-version }}
+      if: ${{ !cancelled() && runner.os != 'Windows' }}
+      shell: bash
+      run: ./tests/fetch_cc-test-reporter || true ;
     - name: Generate Coverage for py3.9 on ${{ matrix.os }}
-      if: ${{ runner.python-version }} == "3.9"
+      if: ${{ runner.python-version == '3.9' }}
```

actionlint (1.7.4)

property "python-version" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)